docs/fleet/provision.md

```diff
@@ -1,278 +1,210 @@
 
-- To start the Viam Agent:
+End users receive a machine, and use either a captive web portal or mobile app to complete the machine setup.
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl start viam-agent
-  ```
+One option is to use the [Viam mobile app](/fleet/control/#control-interface-in-the-viam-mobile-app).
+The Viam mobile app allows end users to create a new machine in the app, and `agent-provisioning` will then install `viam-server` and run it with a provided configuration.
 
-- To stop the Viam Agent:
+To add your branding, you can build your own mobile app and use the [Flutter SDK](https://flutter.viam.dev/viam_protos.provisioning.provisioning/ProvisioningServiceClient-class.html) or the [TypeScript SDK](https://github.com/viamrobotics/viam-typescript-sdk/blob/main/src/app/provisioning-client.ts) to connect to `viam-agent` and provision your machines.
 
-  {{< alert title="Alert" color="note" >}}
-  When you stop the Viam Agent, the agent will stop `viam-server` as well.
-  {{< /alert >}}
-
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl stop viam-agent
-  ```
-
-- To restart the Viam Agent:
-
-  {{< alert title="Alert" color="note" >}}
-  When you restart the Viam Agent, the agent will restart `viam-server` as well.
-  {{< /alert >}}
-
-  ```sh {class="command-line" data-prompt="$"}
-  sudo systemctl restart viam-agent
-  ```
-
-- To completely uninstall the Viam Agent and `viam-server`, run the following command:
+If you are not using Flutter or TypeScript and would like to use provisioning, please [contact us](mailto:support@viam.com).
 
-  ```sh {class="command-line" data-prompt="$"}
-  sudo /bin/sh -c "$(curl -fsSL https://storage.googleapis.com/packages.viam.com/apps/viam-agent/uninstall.sh)"
-  ```
+Regardless whether provisioning happens through the captive web portal or a mobile app, the process follows the same pattern:
 
-  This command uninstalls Viam Agent, `viam-server`, the machine configuration file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
+{{<video webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with viam-agent." poster="/platform/provisioning-demo.jpg" class="alignright" max-width="400px" style="margin-left: 2rem">}}
 
-  For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
+1. When a machine that has `viam-agent` installed and `agent-provisioning` configured comes online, `agent-provisioning` creates a WiFi hotspot.
 
-### View Viam Agent logs
+   - The [`agent-provisioning` configuration](#configuration) is at <file>/etc/viam-provisioning.json</file.
+   - The [`viam-agent` configuration](/configure/agent/#configuration), if provided, is at <file>/etc/viam.json</file> alongside the rest of the machine configuration, including which fragments to use (if any).
```

Comment:
This is not correct. The machine configuration (including fragments) is in the cloud/App. /etc/viam.json contains the machine credentials to connect TO the cloud.